src/components/WwwFrame/EndOfYearAppealBanner/AppealBanner.vue

```diff
@@ -1,22 +1,17 @@
 <template>
-	<div class="appeal-banner-wrapper">
+	<div class="appeal-banner-wrapper" v-if="showAppeal">
```

Comment:
@BoulderBrains We had lost the visibility check somewhere along the way. This new variable ensures at least 1 of the Appeal features is turned on and doesn't show if we're on checkout.

Reply:
🆒